pkg/scheduler/actions/common/allocate.go

```diff
@@ -30,7 +30,7 @@ func AllocateJob(ssn *framework.Session, stmt *framework.Statement, nodes []*nod
 	nodeSets, err := ssn.SubsetNodesFn(job, tasksToAllocate, nodes)
 	if err != nil {
 		log.InfraLogger.Errorf(
```

Comment:
I think we said it is not a scheduler error and it should be info:
```
log.InfraLogger.V(6).Infof(...)
``

Reply:
We said that if a topology object not found is not a scheduler error, but a job error. I agree.

But now if its job error (Topology typo / Not found domain).

If SubsetNode fails now, it is a scheduler error